tracing-journald/src/lib.rs

```diff
@@ -126,7 +126,7 @@ where
         let span = ctx.span(id).expect("unknown span");
         let mut buf = Vec::with_capacity(256);
 
-        let depth = span.parents().count();
+        let depth = span.scope().skip(1).count();
```

Comment:
Not sure that this is actually worth doing, but incrementing `depth` would technically be a breaking change...